FrontEnd/src/components/SignUp/components/signup/signup-form/RulesModal.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './RulesModal.module.css';
+import rulesText from './Rules_text';
+
+const RulesModal = ({ closeModal }) => {
+  return (
+    <div className={styles.modal}>
+      <div className={styles['rules-modal']}>
+        <h2 className={styles['rules-modal__title']}>{rulesText.title}</h2>
+        {rulesText.content.map((item, index) => {
+          if (item.type === 'paragraph') {
+            return <p key={index}>{item.text}</p>;
```

Comment:
I think it might be ok to use the index as a key in this specific case.
But FYI its an anti-pattern https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067

Reply:
I used id in rules-text.js and added an alternative UUID generator if id is absent